src/eventHandler.js

```diff
@@ -10,10 +10,12 @@ export default superclass => class extends superclass {
 	 */
 	_holdHandler(e) {
 		const conf = this._conf;
+		const touch = conf.touch;
 		const holdPos = e.pos.flick;
 
-		conf.touch.holdPos = holdPos;
-		conf.touch.holding = true;
+		touch.holdPos = holdPos;
+		touch.holding = true;
```

Comment:
eg.Axes v2.2.0 pass holding and isTrusted properties.
how about using these properties?

Reply:
May be it's good, but using prop value from the dependency will make deep attachment and is likely have unknown error if the dependency module changes its behavior.